code/game/machinery/doors/airlock.dm

```diff
@@ -1287,6 +1287,10 @@
 		prying_so_hard = FALSE
 		return
 
+	if(SEND_SIGNAL(tool, COMSIG_JAWS_OF_LIFE_FORCE_OPEN_AIRLOCK, user, src) & COMPONENT_JAWS_DO_NOT_ALLOW)
```

Comment:
This proc isn't just called by jaws of life, right? This signal might be misleadingly named
(Maybe should be something like `COMSIG_ITEM_FORCING_OPEN_AIRLOCK`)
(Also it seems like tool can be null? But the line right above doesn't null check for tool either...)

Reply:
https://github.com/tgstation/tgstation/blob/2075a27dc62d432de940a66ed4ae20bc9aae4eb2/code/modules/transport/tram/tram_doors.dm#L226-L234

Ah, here we are. Instance of it being null.